Lib/Inc/error_log.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file error_log.h
+ */
+
+/*
+ *  Created on: Feb 12, 2022
+ *      Author: niuslar
+ */
+
+#ifndef ERROR_LOG_H_
+#define ERROR_LOG_H_
+
+#include <string>
+#include "uart_com.h"
+
+class CLog
+{
+public:
```

Comment:
ctor and dtor should be first methods.
``` 
class CClassName
{
  CClassName(...); //ctor - always comes first unless it's special cases like Singleton class
  ~CClassName(...); //dtor - always comes after ctor.
// public methods - these present the most interest to the user of the class, so they should be high up.

// protected methods - if they exist

// private methods - may not even declare them in a .h file, but people usually do.

// public members - if they exist they are secodn most important information for the user.

// protected members - if they exist

// private members - at the bottom because least interesting to the user.
```